include/mbgl/map/map.hpp

```diff
@@ -79,8 +79,8 @@ class Map : private util::noncopyable {
     void setClasses(const std::vector<std::string>&);
     std::vector<std::string> getClasses() const;
 
-    void setDefaultTransitionDuration(Duration = Duration::zero());
-    Duration getDefaultTransitionDuration();
+    void setDefaultTransitionDuration(const Duration& = Duration::zero());
+    const Duration getDefaultTransitionDuration() const;
```

Comment:
Good point, I guess that went through on my regular expression :ghost: will remove that.